meta-flex-distro/recipes-core/packagegroups/packagegroup-base.bbappend

```diff
@@ -2,6 +2,6 @@
 # SPDX-License-Identifier: MIT
 # ---------------------------------------------------------------------------------------------------------------------
 
-FILESEXTRAPATHS:prepend:feature-sokol-flex-staging := "${THISDIR}/${BPN}:"
+RDEPENDS:packagegroup-base-bluetooth:append:flex-os = "${@bb.utils.contains('DISTRO_FEATURES', 'pulseaudio', 'pulseaudio pulseaudio-server', '', d)}"
```

Comment:
This is not only the distro change. You are doing some cleanup as well. I think its explaination should be added

Reply:
Nope, there is no other cleanup other than removal of Sokol keyword which is mentioned in the description. This diff is just because git is confusing file renaming of different files.